src/address.ts

```diff
@@ -174,10 +168,9 @@ export class Address {
   //
 
   /**
-   * Returns a random localized street address
+   * Returns a random localized street address.
```

Comment:
```suggestion
   * Generates a random localized street address.
```
(not sure about this one)